storage/src/main/java/fk/prof/storage/Callback.java

```diff
@@ -0,0 +1,9 @@
+package fk.prof.storage;
+
+/**
+ * @author gaurav.ashok
+ */
+@FunctionalInterface
+public interface Callback {
```

Comment:
It does not have relation to storageEvent. I just needed a generic interface for a callback.